bindings/c/local.mk

```diff
@@ -26,6 +26,10 @@ fdb_c_LIBS := lib/libfdbclient.a lib/libfdbrpc.a lib/libflow.a
 fdb_c_tests_LIBS := -Llib -lfdb_c
 fdb_c_tests_HEADERS := -Ibindings/c
 
+ifndef __TLS_DISABLED__
```

Comment:
I'd much prefer TLS_DISABLED for the Makefile and #define.  Two leading underscore names are technically reserved by the C/C++ standard.